modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure the {PM-kepler} to use the Redfish as the source for platform power.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure {PM-kepler} instance’s `.spec.exporter.redfish` specifying the mandatory `secretRef` and optional `probeInterval` and `skipSSLVerify`.
++
+.Example output
+[source,yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:  ... 
+    redfish:
+      secretRef: <name of the secret> required <1>
+      probeInterval: 60s <2>
+      skipSSLVerify: false <3>
+----
+<1> Specifies the name of the secret that contains the credentials for accessing the Redfish server. This field is required.
```

Comment:
I think frontloading the callouts with "Required/no required" would help customers instantly know what is mandatory before they read a description.

Reply:
Rephrased this to:
<1> Required: Specifies the name of the secret that contains the credentials for accessing the Redfish server.
<2> Optional: Controls the frequency at which the power information is queried from Redfish. The default value is `60s`.
<3> Optional: Controls if {PM-kepler} skips verifying the Redfish server certificate. The default value is `false`.